conf.yaml

```diff
@@ -1363,14 +1363,34 @@ contents:
                         repo:   apm-agent-rum-js
                         path:   docs
           -
```

Comment:
@Titch990 How about changing L1215:
```
        title:      Observability: APM, Infrastructure, Logs, and Uptime
```

To this:
```
        title:      Observability: APM, Logs, Metrics, and Uptime
```

Based on this image:
![01-top-level](https://user-images.githubusercontent.com/5618806/67301891-e5479b80-f4a4-11e9-8cc7-54ba8bcaa6f0.png)

Reply:
Fair comment. I have a separate task to change Infrastructure to Metrics throughout, so I wasn't too bothered about finding them all yet, but it does make sense to pick this one off now.